ios/Classifier/VCPTaxonomy.m

```diff
@@ -174,14 +175,26 @@ - (NSDictionary *)aggregateScores:(MLMultiArray *)classification currentNode:(VC
 
         if ([leafScore floatValue] >= self.taxonomyRollupCutoff) {
             allScores[node.taxonId] = leafScore;
+        } else {
+            self.excludedLeafScoreSum += leafScore.floatValue;
```

Comment:
I am not sure if this is the Obj-C way of doing things.

Reply:
looks good